src/main.rs

```diff
@@ -782,14 +783,40 @@ struct Cli {
     /// Used by child process such that supervisor parent process can control it
     #[arg(long)]
     child_process: bool,
+    /// Show uninstall dialog and delete configuration and logs
+    #[arg(long)]
+    uninstall: bool,
     /// The rest of the arguments that will be sent to GTK4 as is
     #[arg(raw = true)]
     gtk_arguments: Vec<String>,
 }
 
 impl Cli {
     fn run(self) -> ExitCode {
-        if self.child_process {
+        if self.uninstall {
+            if cfg!(windows)
+                && MessageDialog::new()
+                    .set_type(MessageType::Info)
+                    .set_title("Uninstall")
+                    .set_text("Delete Space Acres configuration and logs for all users?")
+                    .show_confirm()
+                    .is_ok_and(|x| x)
```

Comment:
Also this is more idiomatic way to handle result:
```suggestion
                    .unwrap_or_default()
```